apps/backend/src/resources/fixedWidthRecord/fixedWidthRecord.ts

```diff
@@ -69,7 +72,7 @@ export class FixedWidthRecordService {
    * @param clz the class/constructor

```

Comment:
I realize this is not a part of the diff, but I am wondering if we should include these jsdoc annotations throughout the codebase? This could be useful later for generating documentation. Although, I am not sure if we need/want both jsdoc and open api or if exclusively using one of these would be best?